Documentation/configuring-reporting-operator.md

```diff
@@ -1,18 +1,20 @@
 # Configuring reporting-operator
 
 The reporting-operator is responsible for collecting data from Prometheus, storing the metrics in Presto, running report queries against Presto, and exposing their results via an HTTP API.
+
 Configuring the operator is primarily done within a `MeteringConfig` Custom Resources's `spec.reporting-operator.spec` section.
 
 ## Prometheus connection
 
-Depending on how you installed Metering, the default Prometheus URL varies.
-If you installed for Openshift then the default assumes Prometheus is available at `https://prometheus-k8s.openshift-monitoring.svc:9091/`.
-Otherwise it assumes that your Prometheus service is available at `http://prometheus-k8s.monitoring.svc:9090`.
-If you're not on Openshift and aren't using [kube-prometheus][kube-prometheus], then you will need to override the `reporting-operator.config.prometheus.url` configuration option.
+When deploying Metering on an Openshift cluster, the reporting-operator defaults to using the Thanos Querier interface in the openshift-monitoring namespace: `https://thanos-querier.openshift-monitoring.svc:9091/`.
+
```

Comment:
Is there a particular min version OpenShift that this is true for?

Reply:
Actually just read the PR summary, so 4.3, that's the min version of Openshift we really support, right? So I guess no need to mention the version?

Reply:
master docs doc current release == 4.5, so all set.